.github/workflows/README.md

```
@@ -0,0 +1,44 @@
+- A PAT (Personal Access Token) with `repo` scope stored as `MONOREPO_DISPATCH_TOKEN` in the flowglad-node repository secrets
+
+### What it does
+
+1. Replaces `@flowglad/node` in all SDK packages with the Stainless tarball URL
+2. Installs dependencies
+3. Builds all packages
+4. Runs the test suite
+
+### Tarball URL format
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)